GREEDY/Data/ShopDataModel.cs

```diff
@@ -10,6 +10,7 @@ public class ShopDataModel
         [Required]
         public string Name { get; set; }
         public string Location { get; set; }
+        public string SubName { get; set; }
```

Comment:
Can you explain what is SubName?

Reply:
I add comment about subname in shop method. Subname is like a name how that shop is named in formal way (like UAB MAXIMA, UAB PALINK = iki) and other. Sometimes name as we know not the same as in formal documents